src/templates/automation-analytics/Components/ReportComponents/Standard/Components/TemplateExplorerExpandedRow/index.tsx

```diff
@@ -3,6 +3,7 @@ import { Flex, FlexItem, Grid, GridItem } from '@patternfly/react-core';
 import { ExpandableRowContent, Td, Tr } from '@patternfly/react-table';
 
 import Breakdown from './BreakdownChart';
+// import CategoryCount from './BreakdownChart';
```

Comment:
Why we can't use this value?

Reply:
we are! it was being defined where it wasn't used, just moved it to this file instead :) 

I'll clean it up